docs/examples/ensemble-dimensions.ipynb

```diff
@@ -0,0 +1,308 @@
+   "source": [
+    "## Intersection\n",
+    "\n",
+    "A first way of aggregating the scores is to consider each score along a dimension as being an expert. The idea is then to take the __intersection__ of the experts so that the predicted changepoints are the changepoints where all the experts are confident. This method is useful when the user is interested in collecting the changepoints that have been detected on all dimensions at the same timestep.\n",
+    "\n",
+    "In the following cell, the intersection procedure correctly predicts the only common changepoint between the two dimensions. The aggregated score shows clearly the only changepoint to predict."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bkps_intersection = [bkps[1], n_samples]\n",
+    "n_bkps_intersection = 1\n",
+    "\n",
+    "# intersection aggregation\n",
+    "algo.score = min_(minmax(scores))\n",
+    "bkps_intersection_predicted = algo.predict(n_bkps=n_bkps_intersection)\n",
+    "\n",
+    "# For display purpose\n",
+    "appended_intersection_scores = np.append(\n",
+    "    np.ones(window_size // 2) * float(\"inf\"), algo.score\n",
+    ")\n",
+    "\n",
+    "fig, (ax,) = rpt.display(\n",
+    "    appended_intersection_scores, bkps_intersection, bkps_intersection_predicted\n",
+    ")\n",
+    "_ = ax.set_title(\"Aggregated score for intersection purpose\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Union\n",
+    "\n",
+    "Another way of aggregating the scores is to take the __union__ of the experts so that the predicted changepoints are the changepoints where at least one expert is confident. This method is useful when the user is interested in collecting all the changepoints that are present along all dimensions.\n",
+    "\n",
+    "In the following cell, the union procedure correctly predicts all the changepoints. The aggregated score shows 5 clear peaks from which to take the changepoints."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bkps_union = bkps\n",
+    "n_bkps_union = 5\n",
+    "\n",
+    "# union aggregation\n",
+    "algo.score = max_(minmax(scores))\n",
+    "bkps_union_predicted = algo.predict(n_bkps=n_bkps_union)\n",
+    "\n",
+    "# For display purpose\n",
+    "appended_union_scores = np.append(np.ones(window_size // 2) * float(\"inf\"), algo.score)\n",
+    "\n",
+    "fig, (ax,) = rpt.display(appended_union_scores, bkps_union, bkps_union_predicted)\n",
+    "_ = ax.set_title(\"Aggregated score for union purpose\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "\n",
+    "This example shows a way of crafting a changepoint detection algorithm at the scale of the dimensions. This is a finer scale than the usual one: the signal scale.\n",
```

Comment:
```suggestion
    "Rather than using \"regular\" cost functions that needs to aggregate all dimensions in the first place, this example introduces a way of crafting a changepoint detection algorithm based on a smaller dimension level.\n",
```